docker-compose.airflow.yml

```diff
@@ -0,0 +1,155 @@
+version: '2'
+services:
+  redis:
+    image: redis:3-alpine
+
+  nginx: 
```

Comment:
I think that the `extra_hosts` are OK to keep across the board. Interaction with the Airflow UIs locally always happens with port forwarding directly to the service unproxied by Nginx.